doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -135,6 +135,20 @@ Additional scenario XML elements:
   Specify whether ACRN hypervisor image can be booted using multiboot2 protocol.
   If not set, GRUB's multiboot2 is not available as a boot option.
 
+``RDT_ENABLED`` (a child node of ``FEATURES/RDT``):
+  Specify whether enable Resource Director Technology(RDT) allocation feature.
+  If HW board does not support this feature, UI would have only one 'n' optional.
+
+``CDP_ENABLED`` (a child node of ``FEATURES/RDT``):
+  Specify whether enable Code and Data Prioritization(CDP). CDP is an extension
+  of CAT.
+  If HW board does not support RDT or select 'n' for ``RDT_ENABLED``, UI would
+  have only one 'n' optional.
+
+``CLOS_MASK`` (a child node of ``FEATURES/RDT``):
+  Specify cache masks value, make sure the filled value is continuous bits.
+  If HW board does not support RDT, then no ``CLOS_MASK`` settings.
+
```

Comment:
Specify cache capacity bitmask for the CLOS, only continuous '1' bits are allowed.
The value will be ignored when hardware does not support RDT.